ctf/check.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
like I said, what's wrong with a single 'batch' lua script that would work on all system?